lib/utils.js

```diff
@@ -24,14 +24,15 @@ exports.findIndexOf = findIndexOf;
 
 var traverse = require('traverse');
 var assert = require('assert');
+var g = require('strong-globalize')();
 var Promise = require('bluebird');
 
 function safeRequire(module) {
   try {
     return require(module);
   } catch (e) {
-    console.log('Run "npm install loopback-datasource-juggler ' + module +
-      '" command to use loopback-datasource-juggler using ' + module +
+    g.log('Run "npm install {{loopback-datasource-juggler}} ' + module +
+      '" command to use {{loopback-datasource-juggler}} using ' + module +
       ' database engine');
```

Comment:
You need to replace with %s and move the variable to the end separated by commas.